### ISSUE TITLE
Use tag style for font variants list

### ISSUE THREAD

**Issue by NathanBnm** [aimproxy/dotfonts#7]
It would be great to use tags to improve font variants (regular, italic, bold, 700, 400...) styling.

I'm thinking about the "Planner" app

**Comment by aimproxy** [comment]
Yeah I see whats you think about, I allready think about tags instead of string builder! Thx
